setup-tooling.sh

```diff
@@ -0,0 +1,3 @@
+#!/bin/sh
+echo "Installing prettier"
+npm install --save-dev --save-exact prettier
```

Comment:
Is this necessary; 
It should be in our package.json already, I believe.
Why don't we just commit that.

Yes, you have to run `npm install` still, but you would have to in order to run anything anyway?

I am just confused, I guess 😅

Reply:
So we have 2 package.json files.

One inside the root of the project containing prettier as a dependency. Why? Not sure, I didn't add it. I assume for the sake of convenience? 
One inside ./src that contains all of our frontend dependencies, also including prettier

Reply:
I have wanted to clean that up for a while, but not taken the time.

Having two package.json is a no-go in my mind, and only adds confuzzlement.
Let's do that later, and get this change in now, then 😄